server/lib/nl_data_spec.py

```diff
@@ -149,43 +158,17 @@ def compute(query_detection: Detection, context):
         expanded_svs.extend(expanded_svgs[sv])
     else:
       selected_svs.append(sv)
-  # relevant_svs_df = _filtered_svs_df(svs_df)
-  # relevant_svs = relevant_svs_df['SV'].values.tolist()
-
-  # Get related SVGs and all info.
-  # svgs_info = _related_svgs(relevant_svs, all_relevant_places)
-
-  # Get useful sv2name and sv2definitions.
-  # sv_maps = _sv_definition_name_maps(svgs_info, relevant_svs)
-  # sv2name = sv_maps["sv2name"]
-  # sv2definition = sv_maps["sv2definition"]
-
-  # Get SVGs into peer buckets.
-  # peer_buckets = _peer_buckets(sv2definition, relevant_svs)
-
-  # Produce Chart Config JSON.
-  # chart_config = _chart_config(place_dcid, main_place_type, main_place_name,
-  #                              child_places_type, highlight_svs, sv2name,
-  #                              peer_buckets)
 
-  # message = ParseDict(chart_config, subject_page_pb2.SubjectPageConfig())
-
-  # This is a new try to extend svs to siblingins. This is to extend the
-  # stat vars "a little bit"
-
-  context_sv = None
-  if context and context['debug'] and context['debug']['data_spec']:
-    context_sv = context['debug']['data_spec']['primary_sv']
+  use_context_sv = False
+  if not selected_svs and context_sv:
+    selected_svs = [context_sv]
```

Comment:
should this be highlight_svs, so we use context when there are only poor matches?

Reply:
from here downward, only selected_svs are used? so here would directly use context_sv for further processing.